Sming/SmingCore/HardwareSerial.h

```diff
@@ -254,6 +254,12 @@ class HardwareSerial : public Stream
 	 */
 	size_t indexOf(char c);
 
+	/*
+	 * @brief Returns a pointer to the internal uart object. Use with care.
+	 * @retval pointer to uart_t
+	 */
+	uart_t* getUart() { return uart; }
```

Comment:
@tius2000 We are enforcing coding standard for files under `samples` and `SmingCore`.  Please make sure to install clang-format and then run `make cs`. For more information read the `Coding Style Rules` section in the [release notes](https://github.com/SmingHub/Sming/releases/tag/3.6.1).